src/integrationTest/resources/TestProject/build.gradle

```diff
@@ -0,0 +1,34 @@
+buildscript {
+  repositories {
+    maven {
+      url uri('libs')
+    }
+    //flatDir dirs: "C:/Users/i_am_/workspace/umple.gradle/build/libs"
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
+  dependencies {
+    // Use the umple jar from the main umple.gradle project
+    classpath files('../../../../libs/umple-latest.jar')
```

Comment:
Make the parent project copy this JAR somewhere local to this subproject. This path is too long and dangly.